Sources/Testing/Support/Versions.swift

```diff
@@ -141,23 +142,50 @@ var targetTriple: String? {
 
 /// A human-readable string describing the Swift Standard Library's version.
 ///
-/// This value's format is platform-specific and is not meant to be
-/// machine-readable. It is added to the output of a test run when using
-/// an event writer.
+/// This value is unavailable on some earlier Apple runtime targets. On those
+/// targets, this property has a value of `5.0.0`.
 ///
 /// This value is not part of the public interface of the testing library.
-let swiftStandardLibraryVersion: String = {
-  if #available(_swiftVersionAPI, *) {
-    return String(describing: _SwiftStdlibVersion.current)
+let swiftStandardLibraryVersion: VersionNumber? = {
```

Comment:
It's not clear to me under what conditions this would be `nil`, so I'm curious why it's Optional?

Reply:
The stdlib function we call has limited availability.